src/Cpp/1-getting-started/11-HelloTriangle/HelloTriangle.cpp

```diff
@@ -51,7 +54,10 @@ bool HelloTriangleApplication::Initialize()
         0u, 1u, 2u
     };
     _triangle = std::make_unique<StaticMesh>();
-    if (!_dxContext->MakeStaticMesh(std::move(vertices), std::move(indices), _triangle.get()))
+    if (!_dxContext->MakeStaticMesh(
```

Comment:
not a fan of moving into MakeStaticMesh and then in there moving it back into _triangle, maybe change the API and do:
```cpp
_triangle.vertices = {
         0.0f,  0.5f, 0.0f,
         0.5f, -0.5f, 0.0f,
        -0.5f, -0.5f, 0.0f,
    };
    _triangle->indices =  {
        0u, 1u, 2u
    };
   _triangle->MakeStaticMesh() ;
   ```
   instead? 
   
   Or just perhaps accept the potential copy (or rely on copy-elision) and give normal const&'s instead

Reply:
I agree, I will push a fix right now